apps/nextjs-website/src/lib/header.ts

```diff
@@ -60,6 +60,21 @@ export const removeHomepageSlugFromMenu = (
             })),
           },
         ],
+        drawerMenuTitle: 'text',
+        ctaTitle: 'text',
+        ctaButtonText: 'text',
+        ctaHref: 'text',
+        ctaBodyText: 'text',
+        drawerCardsData: [
+          {
+            title: 'Cittadini',
+            subtitle:
+              'Accedi come persona fisica, libero professionista o ditta individuale',
+            stackIcon: 'People',
+            buttonText: 'Accedi',
+            href: '/',
+          },
+        ],
```

Comment:
Parse this data from what's been input, why are hardcoded values being assigned here!?